src/mame/taito/tnzs.cpp

```diff
@@ -750,15 +749,14 @@ void kageki_state::csport_w(uint8_t data)
 		{
 			// play samples
 			m_samples->start_raw(0, m_sampledata[data].get(), m_samplesize[data], 7000);
-			sprintf(mess, "VOICE:%02X PLAY", data);
+			LOG("VOICE:%02X PLAY", data);
```

Comment:
I assume we don't want/need to comment out LOG?
Previously popmessage was commented out.

Reply:
You don’t need to comment out `LOG` – it’s disabled by default, but it compiles the expressions to avoid rot (you need to `#define VERBOSE 1` or `#define VERBOSE LOG_GENERAL` to enable it).

Reply:
Great thanks